include/onnx-mlir/Compiler/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 install(FILES OMCompilerTypes.h DESTINATION include/onnx-mlir/Compiler)
+install(FILES OMCompilerMacros.h DESTINATION include/onnx-mlir/Compiler)
```

Comment:
Are all the headers being installed now? In the PR I had sent, I had added a few headers that were previously missing to the install. Can you do that too? https://github.com/onnx/onnx-mlir/pull/1262

Reply:
Done. I pulled them from your PR.